### DISCUSSION TITLE
Carousel update (swiper js)

### DISCUSSION THREAD

**Discussion by vitalijalbu** [saadeghi/daisyui#1243] [Ideas / feature requests]
I can suggest swiper js for carousel, it's very light & flexible

**Comment by saadeghi** [comment]
Thank you.  
The carousel from daisyUI is just a CSS-only solution and of course it can not compete with any JS plugin. JS plugins are more accessible and have many more options.

I suggest everyone to use the only daisyUI components they need, and wherever a JS solution offers more, use that JS plugin (such as carousels, modals, etc.)
